lorem.html

```diff
@@ -18,7 +18,7 @@
     <link rel="stylesheet" href="styles/main.css">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Proza+Libre:wght@500&display=swap" rel="stylesheet">
     <style>
-        image {
+        main img {
```

Comment:
Wrong element name
Also, make sure you're being specific enough with your selectors